File: PYTHON PROGRAMMING/Test22.py
## 22 ARITHMETIC OPERATION CALCULATOR IN PROGRAMMING
def func(num1, num2):
    while True:
        choice = input("Enter Your Choice (+, -, *, /, %, exit): ")

        match choice:
            case "+":
                print(f"{num1} + {num2} = {num1 + num2}")

            case "-":
                print(f"{num1} - {num2} = {num1 - num2}")

            case "*":
                print(f"{num1} * {num2} = {num1 * num2}")

            case "/":
                if num2 == 0:
                    print("Division by zero is not allowed")
                else:
                    print(f"{num1} / {num2} = {num1 / num2}")

            case "%":
                if num2 == 0:
                    print("Modulo by zero is not allowed")
                else:
                    print(f"{num1} % {num2} = {num1 % num2}")

            case "exit":
                print("Exiting calculator...")
                break

            case _:
                print("Invalid choice! Please try again.")

num1 = int(input("Enter first number: "))
num2 = int(input("Enter second number: "))
func(num1, num2)

File: TYPESCRIPT PROGRAMMING/Test22.ts
// 22. ARITHMETIC OPERATION CALCULATOR IN PROGRAMMING
const func = (ip1: number, ip2: number, choice: string): String => {
    while(true) {
        switch(choice) {
            case "+":
                return `${ip1} + ${ip2} = ${ip1 + ip2}`;
            case "-":
                return `${ip1} - ${ip2} = ${ip1 - ip2}`;
            case "*":
                return `${ip1} * ${ip2} = ${ip1 * ip2}`;
            case "/":
                return `${ip1} / ${ip2} = ${ip1 / ip2}`;
            case "%":
                return `${ip1} % ${ip2} = ${ip1 % ip2}`;
            default:
                return `Invalid Operation.`
        }
    }
}
console.log(func(125, 23, "+"));
console.log(func(125, 23, "-"));
console.log(func(125, 23, "*"));
console.log(func(125, 23, "/"));
console.log(func(125, 23, "%"));
console.log(func(125, 23, "55"));